alonzo/formal-spec/utxo.tex

```diff
@@ -691,6 +688,10 @@ \subsection{Witnessing}
 We construct the following helper functions :
 
 \begin{itemize}
+  \item $\fun{witsVKeyNeeded}$ is a Shelley function adjusted to include
+  keys that are specified in the $\fun{reqSignerHashes}$ field of the transaction,
+  as well as keys locking the collateral inputs of the transaction
```

Comment:
Is this change related?

Reply:
before, we were unioning just `witsVKEYNeeded` with `reqSignerHashes` in the UTXOW rule, now we add another thing to union with, it became too awkward to do this in the rule instead of the UTXOW function itself